source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml

```
@@ -23,18 +25,43 @@ jobs:
       run: >-
         python3 -m
         build
-        --sdist
-        --wheel
-        --outdir dist/
-        .
-    # Actually publish to PyPI/TestPyPI
-    - name: Publish distribution 📦 to Test PyPI
-      uses: pypa/gh-action-pypi-publish@release/v1
+    - name: Store the distribution packages
+      uses: actions/upload-artifact@v3
       with:
-        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
-        repository-url: https://test.pypi.org/legacy/
+        name: python-package-distributions
+  publish-to-pypi:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
```

This job shouldn't duplicate building, it should just publish.
```suggestion
    name: >-
      Publish Python 🐍 distributions 📦 to PyPI
      and sign them with Sigstore
```